besspin/cyberPhys/ui/hacker-kiosk/src/components/Hack05_InfoAttempt.vue

```diff
@@ -28,18 +27,44 @@
 
 
 <script>
+
+  const electron = require('electron')
+  const ipc = electron.ipcRenderer;
+  
   export default {
     name: 'Hack05_InfoAttempt',
     props: {
     },
     data() {
       return {
-        messages: []
+        messages: [],
+        poller: setInterval(() => { this.pollState() }, 3000)
       }
     },
     mounted() {
+      let vm = this;
+      ipc.on('zmq-results',(event, q) => {
+        q.forEach(item => {
+          console.log("item", item);
+          //TODO: Handle Failure?
+          if(item.function == "hack-info" && item.status == 200) {
+            vm.$router.push({ name: 'hack06_info_exploit' });
```

Comment:
The syntax is:

* Request:  `{func: "function to be called" args: {arguments here, can be an empty dictionary}}`
* Response: `{func: "function that was called" args: {arguments used can be an empty dictionary}, retval: "some-return-string", status:  response-status-code}`
```
Exchange format:
{
    func: "name-of-the-function-to-be-called",
    args: {"dictionary of arguments, can be an empty dictionary}
    resp: {"dictionary of return vaues, can be an empty dictionary}
    retval: "return value, assumed to be a string"
    status: response status (integer)
        200 -OK
        500 - unexpected failure
        501 - func not implemented
}
```

The function calls are not important right now, I can easily add them at the end.